databricks_cli/pipelines/cli.py

```diff
@@ -186,20 +186,24 @@ def list_cli(api_client):
 @provide_api_client
 def reset_cli(api_client, pipeline_id):
     """
-    Resets a delta pipeline by truncating tables and creating new checkpoint folders so data is
+    [Deprecated] Use the "update --full-refresh" command instead.
+
+    Resets a pipeline by truncating tables and creating new checkpoint folders so data is
     reprocessed from scratch.
 
     Usage:
 
     databricks pipelines reset --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: \"reset\" command is deprecated, " +
+               "please use \"start --full-refresh\" command instead")
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).reset(pipeline_id)
+    PipelinesApi(api_client).start_update(pipeline_id, full_refresh=True)
     click.echo("Reset triggered for pipeline {}".format(pipeline_id))
 
 
 @click.command(context_settings=CONTEXT_SETTINGS,
-               short_help='Starts the execution of a delta pipeline run')
+               short_help='[Deprecated] Starts the execution of a pipeline run')
```

Comment:
If possible, try to stay away from the word "execution." For example, "Starts a pipeline run."

Reply:
done, ptal